{
 "pytest_faulthandler":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler._cancel_traceback_dump":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler.pytest_addoption":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler.pytest_configure":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler.pytest_enter_pdb":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler.pytest_exception_interact":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler.pytest_report_header":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler.pytest_runtest_protocol":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 },
 "pytest_faulthandler.timeout_support_available":{
  "__set__":true,
  "elements":[
   "pytest-faulthandler-1.3.0-py27_0",
   "pytest-faulthandler-1.3.1-py34_0",
   "pytest-faulthandler-1.4.1-py36_0",
   "pytest-faulthandler-1.6.0-py36_0"
  ]
 }
}